pkg/executor/sortexec/sort.go

```diff
@@ -763,12 +788,18 @@ func (e *SortExec) initCompareFuncs(ctx expression.EvalContext) error {
 	return nil
 }
 
-func (e *SortExec) buildKeyColumns() {
+func (e *SortExec) buildKeyColumns() error {
 	e.keyColumns = make([]int, 0, len(e.ByItems))
 	for _, by := range e.ByItems {
-		col := by.Expr.(*expression.Column)
-		e.keyColumns = append(e.keyColumns, col.Index)
+		switch col := by.Expr.(type) {
+		case *expression.Column:
+			e.keyColumns = append(e.keyColumns, col.Index)
+		case *expression.Constant:
```

Comment:
add some comment here to explain why constant don't need to add to keyColumns

Reply:
> add some comment here to explain why constant don't need to add to keyColumns

done